app/controllers/ratings_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+class RatingsController < ApplicationController
```

Comment:
Oh, now I've reached your `routes.rb` and I see what you did there.
Still, it's not very readable and easy to understand, is it?

Maybe you can come up with some better options how to implement up-/downvotes in an abstracted away fashion?